Header header
float64 vmax
float64 amax
float64 deltaX
float64 deltaY
float64 deltaZ